kpi/deployment_backends/mock_backend.py

```diff
@@ -17,9 +17,7 @@ class MockDeploymentBackend(BaseDeploymentBackend):
     """
     Only used for unit testing and interface testing.
 
-    defines the interface for a deployment backend.
-
-    # TODO. Stop using protected property `_deployment_data`.
+    Defines the interface for a deployment backend.
```

Comment:
doesn't the base backend define the interface?

Reply:
Indeed